### PR TITLE
Feature/your results page

### PR DESCRIPTION
**Pull request by sam-c-dfe**
# Description

Please include a summary of the changes.

## Ticket number (if applicable) - EYQB-366

# How Has This Been Tested?

Running locally, unit tests and e2e tests

# Screenshots

image

# Checklist:

- [x] My code follows the standards used within this project
- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests (Unit, E2E) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Dfe.EarlyYearsQualification.Web/Views/QualificationDetails/Index.cshtml [hunk 1]
```diff
@@ -12,69 +12,132 @@
...
-        <p class="govuk-body">
-            @Model.Content.BookmarkText
-            <br>
-            <br>
-            <a href="#" class="govuk-link">@Model.BookmarkUrl</a>
+    </div>
+    <div class="govuk-grid-column-full govuk-!-static-margin-top-6">
+        <h2 class="govuk-heading-l" id="ratio-heading">@Model.Content.RatiosHeading</h2>
+        @Html.Raw(Model.Content.RatiosText)
+        <table class="govuk-table">
```
Comment (RobertGHippo) [inline src/Dfe.EarlyYearsQualification.Web/Views/QualificationDetails/Index.cshtml:65]: Question: accessibility. Is this table for layout, or is it a genuine table? If it's for layout should we use something other? If it's really a table, then should it have `<th>` headers? Sonar complains on behalf of text readers.

File: src/Dfe.EarlyYearsQualification.Web/Models/RatioRowModel.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+namespace Dfe.EarlyYearsQualification.Web.Models;
+
+public class RatioRowModel
+{
+    public string LevelText { get; init; } = string.Empty;
+    
+    public bool IsAccepted { get; init; }
```
Comment (RobertGHippo) [inline src/Dfe.EarlyYearsQualification.Web/Models/RatioRowModel.cs:7]: Would `IsApproved` be a better name?

File: src/Dfe.EarlyYearsQualification.Web/Views/QualificationDetails/Index.cshtml (hunk 1 above)
Comment (sam-c-dfe) [reply src/Dfe.EarlyYearsQualification.Web/Views/QualificationDetails/Index.cshtml]: This has now been changed to the summary list

File: src/Dfe.EarlyYearsQualification.Web/Models/RatioRowModel.cs (hunk 2 above)
Comment (sam-c-dfe) [reply src/Dfe.EarlyYearsQualification.Web/Models/RatioRowModel.cs]: This has been changed now :-)

**Review by RobertGHippo — APPROVED** [review]

File: src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs [hunk 3]
```diff
@@ -73,9 +73,103 @@ public async Task<IActionResult> Index(string qualificationId)
...
+        
+        model.RatioRequirements.ApprovedForUnqualified = true;
+    }
+
+    private bool CheckRatio(string propertyToCheck, string ratioName, Qualification qualification)
+    {
+        try
+        {
+            var requirement =
+                qualification.RatioRequirements!.FirstOrDefault(x => x.RatioRequirementName == ratioName);
```
[the remark this answers is not in the document]
Comment (sam-c-dfe) [reply src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs]: Updated

File: src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs [hunk 4]
```diff
@@ -73,9 +73,103 @@ public async Task<IActionResult> Index(string qualificationId)
...
+
+            return (bool)requirement!.GetType().GetProperty(propertyToCheck)!.GetValue(requirement, null)!;
+        }
+        catch
+        {
+            logger.LogError($"Could not find property: {propertyToCheck} within {ratioName} for qualification: {qualification.QualificationId}");
```
[the remark this answers is not in the document]
Comment (sam-c-dfe) [reply src/Dfe.EarlyYearsQualification.Web/Controllers/QualificationDetailsController.cs]: Updated now

File: src/Dfe.EarlyYearsQualification.Web/Models/Content/QualificationDetailsModel.cs [hunk 5]
```diff
@@ -5,11 +5,8 @@ namespace Dfe.EarlyYearsQualification.Web.Models.Content;
...
 {
     public string? FromWhichYear { get; init; }
-    public string? ToWhichYear { get; init; }
+    
     public string? QualificationNumber { get; init; }
-    public string? AdditionalRequirements { get; init; }
-
-    public string BookmarkUrl { get; init; } = string.Empty;
 
     public NavigationLink? BackButton { get; init; }
```
[the remark this answers is not in the document]
Comment (sam-c-dfe) [reply src/Dfe.EarlyYearsQualification.Web/Models/Content/QualificationDetailsModel.cs:12]: Updated

**Pull request merged**